### PR TITLE
Add ActionLink extension method to IUrlHelper

### PR DESCRIPTION
**Pull request by Necronux**
#7807

### CODE REVIEW

File: src/Microsoft.AspNetCore.Mvc.Core/UrlHelperExtensions.cs [hunk 1]
```diff
@@ -201,6 +201,27 @@ public static string Action(this IUrlHelper helper, string action, string contro
...
+        /// <summary>
+        /// Generates an absolute URL for the specified <paramref name="actionName"/> and action
+        /// <paramref name="values"/>, which contains the protocol (such as "http" or "https") and host name from the
+        /// current request.
+        /// </summary>
+        /// <param name="helper">The <see cref="IUrlHelper"/>.</param>
+        /// <param name="actionName">The name of the action that is used to generate URL.</param>
+        /// <param name="values">An object that contains action values.</param>
+        /// <returns>The generated absolute URL.</returns>
+        public static string ActionLink(this IUrlHelper helper, string actionName, object values)
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Mvc.Core/UrlHelperExtensions.cs]: The naming of this is suspect. This naming is the same as the `HtmlHelper.ActionLink` which generates an `<a>` tag.

Comment (dougbu) [reply src/Microsoft.AspNetCore.Mvc.Core/UrlHelperExtensions.cs]: @ayepirate let's go with `AbsoluteAction`

**Comment by mkArtakMSFT** [comment]
Thanks for your effort, @recklessforce. Do you plan to address the comments you've received so we can merge this in?

**Comment by recklessforce** [comment]
@mkArtakMSFT I'm currently busy with something else but I will try to get it done this month

**Comment by mkArtakMSFT** [comment]
@ZeroNightzz, do you still plan to complete this, or should we close it?

**Comment by mkArtakMSFT** [comment]
Closing this PR as it seems this PR went stale.

**Pull request closed without merging**